File: vocab.py
from __future__ import print_function
import sys
from collections import Counter
from operator import itemgetter


def main():
    cut = 10
    counter = Counter()
    with open(sys.argv[1], 'r') as f:
        for line in f:
            for word in line.split():
                counter[word] += 1

    for word, count in sorted(counter.items(), key=itemgetter(1), reverse=True):
        if count > cut:
            print(word, count)


if __name__ == "__main__":
    main()

File: vocab.jl

function vocab(file)
    cut = 10
    counter = Dict{String, Int}()
    open(file, "r") do io
        for line in eachline(io)
            for word in split(line)
                counter[word] = 1 + get(counter, word, 0)
            end
        end
    end

    # Sort and display
    guys = collect(counter)
    sort!(guys, by=(x) -> x[2], rev=true)

    for x in guys
        if x[2] > cut
            println(x[1], '\t',  x[2])
        end
    end
end


vocab(ARGS[1])
